fe/fe-core/src/main/java/org/apache/doris/load/routineload/RoutineLoadJob.java

```diff
@@ -1565,21 +1565,26 @@ private String jobPropertiesToJsonString() {
         jobProperties.put("precedingFilter", precedingFilter == null ? STAR_STRING : precedingFilter.toSql());
         jobProperties.put("whereExpr", whereExpr == null ? STAR_STRING : whereExpr.toSql());
         if (getFormat().equalsIgnoreCase("json")) {
-            jobProperties.put("dataFormat", "json");
+            jobProperties.put(PROPS_FORMAT, "json");
```

Comment:
Will this `PROPS_FORMAT` be overwritten by the key in this.jobProperties?

Reply:
yes，it maybe. But the format should be the same, otherwise there is a BUG.